### DISCUSSION TITLE
deleted

### DISCUSSION THREAD

**Discussion by izmyname** [jarun/nnn#1738] [Ideas]

**Comment by jarun** [comment]
What are the alternatives?

**Comment by izmyname** [comment]
> What are the alternatives?

Depends on what alternatives you mean. Right now - an alternative to get info is to use 'open with' and open a terminal afterwards. If you mean - how a prompt would look like - something like [p]lay/[i]nfo. If you mean something else - sorry, I might've misunderstood your question.

**Comment by jarun** [comment]
No, I meant what are the alternative tools?

**Comment by izmyname** [comment]
> No, I meant what are the alternative tools?

perl-image-exiftool is the only one I can recall from ranger optional deps.

**Reply by jarun** [reply]
Please use a custom command as plugin. Since there are not too many options, the current approach is the best.

**Reply by izmyname** [reply]
I just found out there was an instruction how to use custom commands as plugins. That's exactly what I wanted. Thank you for the answer and for all your work.
